tests/cloudformation/checks/resource/aws/example_SecurityGroupRuleDescription/SecurityGroupRuleDescription-FAILED-3.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
@twmartin Why convert it to json?

Reply:
I'm totally cool with converting this to yaml instead. I had seen that a json test existed in the samples previous to my changes, so I figured perhaps there was some significance to including a json cfn template in a check's unit test examples and therefore I should retain that. However, if there isn't much value in the inclusion of a json example here, I think it would be nice for all the test examples for a particular check to be of the same format.

Reply:
The role of "yaml-ingress-rule-with-no-description" test is now played by `SecurityGroupRuleDescription-FAILED-2.yaml`